.github/workflows/production.yml

```
@@ -39,10 +39,16 @@ jobs:
           aws ecr get-login-password --region ${{ secrets.AWS_REGION }} | docker login --username AWS --password-stdin ${{ secrets.AWS_ACCOUNT_ID }}.dkr.ecr.${{ secrets.AWS_REGION }}.amazonaws.com
 
       - name: Tag Docker image
-        run: docker tag revdojo-keycloak:latest ${{ secrets.AWS_ACCOUNT_ID }}.dkr.ecr.${{ secrets.AWS_REGION }}.amazonaws.com/revdojo-keycloak:latest
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:115: Double quote to prevent globbing and word splitting

(shellcheck)